install_config/syncing_groups_with_ldap.adoc

```diff
@@ -1035,105 +1039,241 @@ users: <5>
 <3> The IP address and host of the LDAP server where this Group's record is stored.
 <4> The name of the Group as specified by the sync file.
 <5> The users that are members of the Group, named as specified by the sync file.
-Note that members of nested groups are included since the group membership was flattened
-by the Microsoft Active Directory Server.
-====
+Note that members of nested groups are included since the group membership was
+flattened by the Microsoft Active Directory Server.
 
+[[sync-ldap-configuration]]
 == LDAP Sync Configuration Specification
 
 The object specification for the configuration file is below.  Note that the different schema
-objects have different fields.  For example, <<v1.ActiveDirectoryConfig>> has no `groupsQuery`
-field whereas <<v1.RFC2307Config>> and <<v1.AugmentedActiveDirectoryConfig>> both do.
+objects have different fields.  For example, xref:sync-ldap-v1-activedirectoryconfig[v1.ActiveDirectoryConfig] has no `groupsQuery`
+field whereas xref:sync-ldap-v1-rfc2307config[v1.RFC2307Config] and xref:sync-ldap-v1-augmentedactivedirectoryconfig[v1.AugmentedActiveDirectoryConfig] both do.
 
+[[sync-ldap-v1-ldapsyncconfig]]
 === v1.LDAPSyncConfig
-:hardbreaks:
-LDAPSyncConfig holds the necessary configuration options to define an LDAP group sync
+
+`LDAPSyncConfig` holds the necessary configuration options to define an LDAP
```

Comment:
@enj Yeah there were enough little style guideline changes needed (xrefs, markup, etc.) that I just ended up going the whole way w/ it. If you think we're gonna be revisiting this a lot going forward, maybe we can do something differently.